### PR TITLE
bundle templates into archive with resources

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by camelburrito** [comment]
@erwinmombay - let me know when you need my review on it.

File: tasks/bundle.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const fs = require('fs-extra');
+const config = require('./config');
+
+
+function collectResources(filepath, html, done) {
+  const filename = path.basename(filepath, '.amp.html');
+  const env = jsdom.env(html, function(err, window) {
+    const ampimgs = window.document.querySelectorAll('amp-img[src]');
+    const imgs = [].slice.call(ampimgs).map(function(el) {
+      const src = el.getAttribute('src');
```
Comment (camelburrito) [inline tasks/bundle.js]: you will also need to parse srcset. I am okay with this for now. as we are not using srcset yet. But pls file a bug against yourself.

File: tasks/bundle.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+const util = require('gulp-util');
+const exec = require('child_process').execSync;
+const jsdom = require('jsdom');
+const through = require('through2');
+const path = require('path');
+const fs = require('fs-extra');
+const config = require('./config');
+
+
+function collectResources(filepath, html, done) {
```
Comment (camelburrito) [inline tasks/bundle.js]: i dont see unminified CSS here. is that hard to do?

Comment (erwinmombay) [inline tasks/bundle.js]: should be easy. ill do it in follow up

File: tasks/bundle.js (hunk 1 above)
Comment (erwinmombay) [inline tasks/bundle.js]: ack

**Pull request merged**